content/recipes/findability/wikiDataAccess.md

```diff
@@ -0,0 +1,54 @@
+# Wikidata Access
+ ````{panels_fairplus}
+:identifier_text: FCB053 
+:identifier_link: 'https://w3id.org/faircookbook/FCB053'
+:difficulty_level: 2
+:recipe_type: hands_on
+:reading_time_minutes: 15
+:intended_audience: bioinformatician, data_scientist, data_engineer
+:has_executable_code: nope
+:recipe_name: Gaining  
+```` 
+## Main Objectives
+
+
+The primary purpose of this recipe is:
+
+> How to gain access to Wikidata
+> What restrictions are in place for a Wikidata account
+> Wikidata and RDF
+> Wikidata and SPARQL
+
+## Requirements
+
+---
+
+## Main Content
+
+### Creating an account
+The first step to gaining access to Wikidata is to register an account at (https://www.Wikidata.org/wiki/Wikidata:Main_Page)
+
+
+### Initial Restrictions
+After creating an account, large-scale edits are restricted until an account has been deemed a trusted account. Fifty edits are required for an account to be considered to be trusted
+
+## Conclusion
+In this recipe, we have gone over:
+1) How to create a Wikidata account
+2) Restrictions of editing
+
+
+## Authors
+
+
+| Name                                                                                                                                                                                                                                       | Orcid                                                                                                                        | Affiliation                           | Type                                                                              |                                                              Elixir Node                                                              | Credit Role
+|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------|---------------------------------------|-----------------------------------------------------------------------------------|:-------------------------------------------------------------------------------------------------------------------------------------:|:----------------:|
+| <div class="firstCol"><a target="_blank" href='https://github.com/'><img class='avatar-style' src='https://avatars.githubusercontent.com/no_github'></img><div class="d-block">Zachary Warnes</div></a>  </div>         | <a target="_blank" href='https://orcid.org/0000-0000-0000-0000'><i class='fab fa-orcid fa-2x text--orange'></i></a> | Maastricht University     | <i class="fas fa-graduation-cap fa-1x text--orange" alt="Academic"></i> | <img class='elixir-style' src='/the-fair-cookbook/_static/images/logo/Elixir/ELIXIR-UK.svg' ></img> | Writing – Review & Editing, Conceptualization
+
+---
```

Comment:
Replace this with the new simpler code as demoed in this patch: https://github.com/FAIRplus/the-fair-cookbook/commit/e1c5b2016c27450efd15f1544bc46e230c161845

Reply:
Your "key" to use is `Zachary` which I added in this PR: https://github.com/FAIRplus/the-fair-cookbook/pull/362/files